setup.cfg

```diff
@@ -0,0 +1,21 @@
+[bumpversion]
+current_version = 0.9.0.0
```

Comment:
should we version this similar to the sdk for making it easy to remember what version of SDK are the APIs tested with?

for e.g. if the current SDK version is 0.9.1.0, then let us call the ucsc_apis also 0.9.1.0

this does not mean we need to make a 1:1 release for every SDK version.. but whenever we do release APIs, it can be versioned as per the latest SDK version.

WDYT?